apps/SageAccountWeb/src/AppInitializer.tsx

```diff
@@ -1,16 +1,16 @@
 import React, { useEffect, useState } from 'react'
-import { AppContextProvider } from 'AppContext'
+import { AppContextProvider } from './AppContext'
 import { Redirect } from 'react-router-dom'
-import { SynapseComponents } from 'synapse-react-client'
```

Comment:
You will see that I avoided using `SynapseComponents` in many files. Importing that object busts tree-shaking, which caused us to import a file that broke the deployed version of the site. I have fixed the deployment issue, so I didn't remove every single import like this. If we want to optimize for smaller builds, then that should be a separate, focused effort.

Reply:
Could you open a ticket to track this idea?  Something like "Investigate if dropping the SynapseComponents object reduces the build size significantly"

Reply:
https://sagebionetworks.jira.com/browse/PORTALS-2655